edgelet/iotedge/src/bundle.rs

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use chrono::{DateTime, Local, NaiveDateTime, Utc};
+use std::fs::File;
+use std::path::Path;
+use zip;
+
+use futures::{Future, Stream};
+use tokio::prelude::*;
+
+use crate::error::{Error, ErrorKind};
+use failure::Fail;
+
+use crate::logs::pull_logs;
+use crate::Command;
+
+use edgelet_core::{LogOptions, LogTail::Num, Module, ModuleRuntime};
```

Comment:
Our existing style is to not import individual enum variants like `LogTail::Num`. Instead import `LogTail` and write `LogTail::Num(0)`